bitpoll/poll/views.py

```diff
@@ -931,7 +931,7 @@ def vote(request, poll_url, vote_id=None):
         'choices_matrix': zip(matrix, choices, comments, choice_votes, events),
         'choices': current_poll.choice_set.all(),
         'choices_matrix_len': len(choices),
-        'values': current_poll.choicevalue_set.filter(deleted=False).all(),
+        'values': current_poll.choicevalue_set.filter(deleted=False).all().order_by('-weight', 'title'),
```

Comment:
```suggestion
        'values': current_poll.choicevalue_set.filter(deleted=False).order_by('-weight', 'title'),
```
`all()` should actually not be necessary here (and previously it was not necessary either).